cli_reference/kn-cli-tools.adoc

```diff
@@ -0,0 +1,44 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="kn-cli-tools"]
+= Knative CLI (`kn`) for use with OpenShift Serverless
+:context: kn-cli-tools
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+`kn` enables simple interaction with Knative components on {product-title}.
+
+You can enable Knative on {product-title} by installing {ServerlessProductName}. For more information, see the documentation on xref:../serverless/serverless-getting-started.adoc#serverless-getting-started[Getting started with {ServerlessProductName}].
+
+[NOTE]
+====
+{ServerlessProductName} cannot be installed using the `kn` CLI. A cluster administrator must install the {ServerlessOperatorName} and set up the Knative components, as described in the _Serverless applications_ documentation for {product-title}.
+====
+
+== Key features
+`kn` is designed to make serverless computing tasks simple and concise, and has the following key features:
+
+* Simplest way to xref:../serverless/serving-creating-managing-apps.adoc#serving-creating-managing-apps[deploy serverless applications] from the command line
+* Full support for managing features of Knative Serving, such as services, revisions, and traffic splitting
+// TODO: add docs about revisions
+* Easily create and manage Knative Eventing components, such as event sources and triggers
++
+[NOTE]
+====
+Knative Eventing is currently available as a Technology Preview feature of {ServerlessProductName}.
+====
+* Create xref:../serverless/knative_eventing/serverless-sinkbinding.adoc#serverless-sinkbinding[sink binding] to connect existing Kubernetes applications and Knative services
+* Flexible plugin architecture, similar to `kubectl`, that can be used for extending `kn`
+// * Easily integrate {ServerlessProductName} with OpenShift Pipelines by using `kn` in an OpenShift Pipelines task
+// TODO: Add integrations later when we have docs about this.
+// TODO: add docs about these deployment types?
+* Easily configure xref:../serverless/knative_serving/configuring-knative-serving-autoscaling.adoc#configuring-knative-serving-autoscaling[autoscaling] parameters for Knative services
```

Comment:
Using adverbs like "easily" and "simply" make it sound like marketing; I'd remove them where possible.